README.md

```diff
@@ -92,4 +92,60 @@ for example. The `-c` tells you the path to the config, and `-o` says where to w
 ## looking at results
 
 The results get saved to the output directory. You can use `wandb sync` to upload the results to some [pretty viewers](https://app.wandb.ai/krisrs1128/glacier_mapping/runs/rsii7qj6?workspace=default).
+
+## pre-processing and slicing
+To the run the preprocessing you need two folders, `img_data` and `vector_data`, which contains the tiffs and the labels respectively. The folders should be organized as illustrated bellow. 
+```
+data\
+  img_data\ # tiff files
+    2010\
+      nepal\
+      bhutan\
+      ...
+    2000\
+      nepal\
+      ...
+  vector_data\ # labels, borders, and test labels
+    2010\
+      nepal\ # nepal labels
+      ....
+    2000\
+      ....
+    basin\
+      data\ #.shp files for test labels
+    borders\
+      nepal\
+        data\ #.shp files to nepal borders
```

Comment:
Could you add links to these vector data? Not here, but somewhere in the readme?